pkg/alibabacloud/api/api.go

```diff
@@ -182,13 +182,26 @@ func (c *Client) GetVPCs(ctx context.Context) (ipamTypes.VirtualNetworkMap, erro
 
 // GetInstanceTypes returns all the known ECS instance types in the configured region
 func (c *Client) GetInstanceTypes(ctx context.Context) ([]ecs.InstanceType, error) {
+	var result []ecs.InstanceType
 	req := ecs.CreateDescribeInstanceTypesRequest()
-	resp, err := c.ecsClient.DescribeInstanceTypes(req)
-	if err != nil {
-		return nil, err
+	req.MaxResults = requests.NewInteger(100)
```

Comment:
Why was 100 chosen? It would be good to have a brief explanation.

Reply:
thank you for your review

The maximum number of entries per page is 100, I add comments

When there are many instance types, some instance type limits can not be queried through the current method, and this problem can be fixed by paging query

Alibaba Cloud API Reference
[1] https://www.alibabacloud.com/help/en/elastic-compute-service/latest/describeinstancetypes

Reply:
Done